### DISCUSSION TITLE
9.0.537

### DISCUSSION THREAD

**Discussion by dansiegel** [PrismLibrary/Prism#3231] [General]
## What's Changed
* Bump Prism version to 8.2 preview by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2502
* Add Support for NavigateTo within EventToCommandBehavior by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2495
* Adds FileOnQ.Prism.Popups.XCT Community Plugin to Readme by @SkyeHoefling in https://github.com/PrismLibrary/Prism/pull/2515
* Update DialogService.cs by @Gineus in https://github.com/PrismLibrary/Prism/pull/2513
* Update IModuleGroupsCatalog by @niimima in https://github.com/PrismLibrary/Prism/pull/2518
* Task: DispatcherQueue property moved across name spaces. WinUI 3 API … by @sigmarsson in https://github.com/PrismLibrary/Prism/pull/2530
* #1978 - Add a few XML docs and correct class names by @Christian-Oleson in https://github.com/PrismLibrary/Prism/pull/2556
* fix: Adjust net5.0 targetframework to fix assets packaging by @jeromelaban in https://github.com/PrismLibrary/Prism/pull/2542
* Changed the access modifier of the AutowireViewModel method to public by @tstephansen in https://github.com/PrismLibrary/Prism/pull/2557
* Add null check for TabbedPage.Current in OnNavTo/From by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2577
* Fixed dialogservice invoking OnNavigatedTo method by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2589
* Prevent NavigationPage & TabbedPage from inheriting parents BindingContext by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2590
* Added CanShowDialog method to IPageDialogService by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2591
* [housekeeping] Automated PR to fix formatting errors by @github-actions in https://github.com/PrismLibrary/Prism/pull/2592
* XmlDoc for WeakDelegatesManager by @mgnslndh in https://github.com/PrismLibrary/Prism/pull/2607
* chore: Update to Uno 4.0 and WinAppSDK 1.0 by @jeromelaban in https://github.com/PrismLibrary/Prism/pull/2628
* fix: Invalid Uno Skia e2e template change by @jeromelaban in https://github.com/PrismLibrary/Prism/pull/2640
* Removed references to .NET Foundation by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2686
* Updated Prism for Uno Platform by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2687
* bump to 9.0 by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2697
* bump dependencies by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2698
* bump to net6 by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2700
* update to form template by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2707
* Update IDialogResult and DialogResult by @niimima in https://github.com/PrismLibrary/Prism/pull/2761
* update for Uno 4.7 by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2808
* Switch to GitHub actions by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2809
* Maui migration by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2810
* Platform View to ViewModel location provider by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2824
* Uno Hosting by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2822
* Ability to select tabs from TabbedPage's TitleView by @mhrastegari in https://github.com/PrismLibrary/Prism/pull/2797
* fix: Navigation inserts page before current page in NavigationPage stack by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2835
* Namespace imports by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2834
* Transient factory by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2836
* add winappsdk workaround targets by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2837
* fix consolidate artifact needs by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2846
* Use instance of IContainerProvider to resolve dependencies by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2847
* Fix MAUI flyout navigation by @berhir in https://github.com/PrismLibrary/Prism/pull/2849
* IActiveAware activates inactive views by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2848
* Migrate Prism.Events to its own package by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2867
* EA add static instance by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2868
* ci: fix consolidation task by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2872
* Fixing runtime warning by @VitalyKnyazev in https://github.com/PrismLibrary/Prism/pull/2876
* Use script file to consolidate resources by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2875
* Update dependencies by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2883
* Move ViewRegistry to Prism.Core by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2886
* Fix Android Back Button by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2889
* Adding Exception Handler by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2887
* Update nuget packages by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2908
* Revert changes - .NET 461 support by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2906
* Adding AsyncDelegateCommand by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2907
* Dialog Consolidation by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2888
* Maui .NET 8 Support by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2919
* Update to dual license by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2942
* Migrate Navigation Parameters to Prism.Core by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2934
* Update ci dotnet version by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2950
* Migrate Regions to core by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2944
* Adding TryRegister by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2960
* Add nullable annotations to commands by @bdovaz in https://github.com/PrismLibrary/Prism/pull/2959
* Use new Prism.Container packages by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2968
* Remove legacy xmlns namespaces by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2969
* updated prism src readme by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/2971
* Uno 5 update by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2974
* fixing release paths by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2975
* Refactoring dependencies for Uno by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2977
* Adding Splash Screen support for Uno by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2979
* Fix DialogService - Setting XamlRoot by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2980
* fix: winappsdk pri duplication by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2986
* Update to Uno 5.0 by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2998
* Update Package Descriptions to include .NET MAUI by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2992
* Removing Uno.Toolkit dependency by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2999
* Add NavigationView RegionAdapter by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3000
* AsyncDelegateCommand enhancements by @dansiegel in https://github.com/PrismLibrary/Prism/pull/2993
* Remove Window dependency in DialogWindow by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3001
* Add AppAction registration helpers. by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3009
* Removing duplicated region interfaces by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3012
* Provide ConfigureWindow for Uno by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3013
* Adding support for C# Markup by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3023
* Update Uno to net8 and update Uno-Check by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3026
* Use net8 for Uno CI build by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3027
* chore: enable net8.0 windows build by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3029
* Adding Linker Metadata by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3038
* BREAKING: Change OnAppStart to CreateWindow by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3041
* Use RootPage Title/IconImageSource when parent is TabbedPage by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3039
* Adding select tab API to INavigationService by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3014
* Maui Cleanup by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3042
* Adding implicit namespaces for DI packages by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3044
* Update NavigationException XML Docs comments. by @niimima in https://github.com/PrismLibrary/Prism/pull/2965
* Cleanup by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3051
* Navigation RemovePageInstruction not working correctly by @lucacivale in https://github.com/PrismLibrary/Prism/pull/3053
* Fixes issue pushing modals by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3055
* feat: adding Forced option for ViewModelLocationBehavior by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3056
* Merge parameters for segments by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3058
* Update uno-check by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3059
* Go back to without popping each page by @niimima in https://github.com/PrismLibrary/Prism/pull/3048
* Add NavigationService helper from the IWindowManager by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3068
* ci: update uno-check by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3062
* Make IPageDialogService a Singleton by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3073
* Fix unhandled dismissal of iOS modals by @Axemasta in https://github.com/PrismLibrary/Prism/pull/3072
* Update skip options for uno-check by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3075
* Update Package ReadMe by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3080
* Fixes Relative Dialog Size by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3082
* Missing XML Docs by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3083
* Uno 5.1 update by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3084
* Fixes for Android Back by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3085
* Code Refactorings by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3086
* [Enhancement] Make it easier to access IDialogService from Singletons by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3094
* Add generic GoBack and remove legacy GoBackTo by @berhir in https://github.com/PrismLibrary/Prism/pull/3101
* Fixing IsRootPage  by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3110
* Updated README link to prismlibrary.com documentation by @DamianSuess in https://github.com/PrismLibrary/Prism/pull/3112
* Adding MAUI Navigation tests by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3119
* Update Select Tab by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3121
* chore: enable nullability for mvvm helpers by @RLittlesII in https://github.com/PrismLibrary/Prism/pull/3131
* chore: enable nullability for ParameterBase by @RLittlesII in https://github.com/PrismLibrary/Prism/pull/3133
* Update RegionNavigationRegistrationExtensions.cs by @jpislas in https://github.com/PrismLibrary/Prism/pull/3145
* Fix double back on Android by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3148
* Update RegionNavigationRegistrationExtensions.cs by @jpislas in https://github.com/PrismLibrary/Prism/pull/3146
* chore: enable nullability for core mvvm by @RLittlesII in https://github.com/PrismLibrary/Prism/pull/3140
* chore: build Prism on macOS by @RLittlesII in https://github.com/PrismLibrary/Prism/pull/3150
* fix: GetCurrentPage for PrismWindow.Page throwing exception by @RLittlesII in https://github.com/PrismLibrary/Prism/pull/3147
* Delay nulling BindingContext on Android by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3151
* Map LayoutViewRegionAdapter to Maui Layout by @lucacivale in https://github.com/PrismLibrary/Prism/pull/3160
* send android app to background when navigating back on the root page by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/3175
* Enable TabbedPage tests by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3176
* Refactor to ArgumentNullException.ThrowIfNull by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3177
* fix: null reference on TabbedPage evaluation by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3180
* fixing Android hardware button by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/3181
* fixed modal navigation by @brianlagunas in https://github.com/PrismLibrary/Prism/pull/3187
* Update packages by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3157
* Add 'AddOrReplace' to IRegionBehaviorFactory by @lucacivale in https://github.com/PrismLibrary/Prism/pull/3169
* Bump 9.1 by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3191
* Update deployments by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3192
* Add release branches to build by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3193
* Bump deps by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3195
* Update Glob by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3197
* Add OfficialRelease flag by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3198
* Update release trigger by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3200
* Use GH docs sample by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3202
* Update public release spec by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3207
* Use Maui version for Modules by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3205
* Dropping Prism.Forms by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3210
* Package updates by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3224
* Add Missing XML Docs comments by @niimima in https://github.com/PrismLibrary/Prism/pull/3220
* Modularity Fix by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3219
* Disable Uno Check by @dansiegel in https://github.com/PrismLibrary/Prism/pull/3229

## New Contributors
* @SkyeHoefling made their first contribution in https://github.com/PrismLibrary/Prism/pull/2515
* @Gineus made their first contribution in https://github.com/PrismLibrary/Prism/pull/2513
* @niimima made their first contribution in https://github.com/PrismLibrary/Prism/pull/2518
* @sigmarsson made their first contribution in https://github.com/PrismLibrary/Prism/pull/2530
* @Christian-Oleson made their first contribution in https://github.com/PrismLibrary/Prism/pull/2556
* @tstephansen made their first contribution in https://github.com/PrismLibrary/Prism/pull/2557
* @mgnslndh made their first contribution in https://github.com/PrismLibrary/Prism/pull/2607
* @mhrastegari made their first contribution in https://github.com/PrismLibrary/Prism/pull/2797
* @berhir made their first contribution in https://github.com/PrismLibrary/Prism/pull/2849
* @bdovaz made their first contribution in https://github.com/PrismLibrary/Prism/pull/2959
* @lucacivale made their first contribution in https://github.com/PrismLibrary/Prism/pull/3053
* @Axemasta made their first contribution in https://github.com/PrismLibrary/Prism/pull/3072
* @DamianSuess made their first contribution in https://github.com/PrismLibrary/Prism/pull/3112
* @RLittlesII made their first contribution in https://github.com/PrismLibrary/Prism/pull/3131
* @jpislas made their first contribution in https://github.com/PrismLibrary/Prism/pull/3145

**Full Changelog**: https://github.com/PrismLibrary/Prism/compare/v8.1.97...9.0.537

<hr /><em>This discussion was created from the release <a href='https://github.com/PrismLibrary/Prism/releases/tag/9.0.537'>9.0.537</a>.</em>

**Comment by DamianSuess** [comment]
This is awesome! I can't wait to upgrade our projects from the `-pre` release to the latest ❤️ The improvements made to Prism over the past couple of years is amazing and provides a cleaner launch-point for projects.

Thank you, Prism team & contributors!
